### ISSUE TITLE
No valid path exists 

### ISSUE THREAD

**Issue by Alexandratz** [dmsl/anyplace#236]
I managed to upload the floor plans, place the PoIs, make the connections between them and mark the corners with x-markers but I am facing some issues:
1) For some points of interests the navigation is being created properly but some others (even if they are connected with oter PoIs), I am getting the "No valid path exists from your position to the PoI selected!" warning, as Indeed the Json response poi list == null.
2) For the most of the routes, even if my marker is placed a few meters from the selected poi, the route is driving first to the  entrance of the building and after to the selected poi.

Could you please assist? Am I doing something wrong?

Building Info:
"buid": "building_a2bcf869-f6bf-4f79-a101-7d4ed3f06976_1523475614370",
        "name": "Πανεπιστήμιο Μακεδονίας",
        "description": "University of Macedonia",
        "coordinates_lat": "40.625214834611135",
        "coordinates_lon": "22.95997901593853"

**Comment by akki12desai** [comment]
how did you use JSON route data to display routes on viewer?

**Comment by Alexandratz** [comment]
Could you please elaborate a bit? I don't get the question.. The routes ere created in Architect..

**Comment by akki12desai** [comment]
yes the POI and routes are created in architect but you can get route information from APIs. in Viewer, you can see the routes by selecting the POIs.

If you use API for route between POIs , you get JSON data of the route. how do you use that JSON data and display the route on the viewer
